src/components/Info.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import info from './icons/info.svg';
+import './Info.css';
+
+export default class Info extends Component {
```

Comment:
Perhaps this comment got lost, but this name should be more explicit -- ie, `InfoIcon`, not just `Info`.

Reply:
Done